pkg/agent/controller/packetsampling/packetin_test.go

```diff
@@ -0,0 +1,215 @@
+package packetsampling
```

Comment:
License is missing.

Reply:
Sorry, can you make this more clear? not sure what's the questions about?  The license one was fixed.

Reply:
other comments were addressed.